index.php

```diff
@@ -0,0 +1,8 @@
+<?php
+		require_once "vendor/autoload.php";
+		
+		use GuzzleHttpClient;
+		
+        $client = new GuzzleHttp\Client();
+        $res = $client->request('GET', 'https://kinoukr.com/4166-pravdyva-istoriya-bandy-kelli.html');
+        var_dump($res);
```

Comment:
It's a bad idea to show debug information to everyone.
By the way, you shouldn't change this file. Please review the requirements and try to follow them.